src/fmu/dataio/_export_item.py

```diff
@@ -824,10 +823,6 @@ def _data_process_object_arrowtable(self):
         meta["bbox"] = None
         logger.info("Process data metadata for ArrowTable... done!!")
 
-    def _fmu_inject_workflow(self):
-        """Inject workflow into fmu metadata block."""
-        self.dataio.metadata4fmu["workflow"] = self.dataio.workflow
```

Comment:
Where is `_fmu_inject_workflow` now, or it it not needed any more?

Reply:
I don't think it is needed (?) As far as I could see, the only function was to insert `self.dataio.workflow` into the fmu-block. But workflow was already there, inserted as part of the `_get_fmu_meta`. Also, while the `workflow` tag was initially inserted correctly (as an object), this function (when called later) overwrote the workflow object with the string.

(Confirmation/second opinion on this welcomed - I may have missed something here)